rmgpy/tools/uncertaintyTest.py

```diff
@@ -56,7 +56,9 @@ def setUpClass(cls):
         cls.uncertainty.database = RMGDatabase()
         cls.uncertainty.database.load(
             data_dir,
-            kineticsFamilies='all',
+            kineticsFamilies=['1,2_shiftC','6_membered_central_C-C_shift','Disproportionation','H_Abstraction',
+                              'Intra_ene_reaction','intra_H_migration','Intra_R_Add_Exo_scission',
+                             'intra_substitutionS_isomerization','R_Addition_MultipleBond','R_Recombination'],
```

Comment:
Really minor, but could you add a space to align the last line?

Reply:
done